src/itest/kotlin/nl/lifely/zac/itest/InformatieobjectenHistorieTest.kt

```diff
@@ -0,0 +1,154 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package nl.lifely.zac.itest
+
+import io.github.oshai.kotlinlogging.KotlinLogging
+import io.kotest.assertions.json.schema.array
+import io.kotest.assertions.json.schema.jsonSchema
+import io.kotest.assertions.json.schema.obj
+import io.kotest.assertions.json.schema.shouldMatchSchema
+import io.kotest.common.ExperimentalKotest
+import io.kotest.core.spec.Order
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.string.match
+import nl.lifely.zac.itest.client.ItestHttpClient
+import nl.lifely.zac.itest.config.ItestConfiguration.DOCUMENT_FILE_TITLE
+import nl.lifely.zac.itest.config.ItestConfiguration.DOCUMENT_IDENTIFICATION
+import nl.lifely.zac.itest.config.ItestConfiguration.DOCUMENT_STATUS_DEFINITIEF
+import nl.lifely.zac.itest.config.ItestConfiguration.DOCUMENT_STATUS_IN_BEWERKING
+import nl.lifely.zac.itest.config.ItestConfiguration.DOCUMENT_UPDATED_FILE_TITLE
+import nl.lifely.zac.itest.config.ItestConfiguration.PDF_MIME_TYPE
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_PDF_FILE_NAME
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_SPEC_ORDER_AFTER_ZAAK_UPDATED
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_TXT_FILE_NAME
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_USER_1_NAME
+import nl.lifely.zac.itest.config.ItestConfiguration.TEXT_MIME_TYPE
+import nl.lifely.zac.itest.config.ItestConfiguration.ZAC_API_URI
+import nl.lifely.zac.itest.config.ItestConfiguration.enkelvoudigInformatieObjectUUID
+import nl.lifely.zac.itest.util.shouldEqualJsonIgnoringExtraneousFields
+
+@OptIn(ExperimentalKotest::class)
```

Comment:
What is this and why do we need it?

Reply:
Needed for `shouldMatchSchema jsonSchema`. Kotest expects to change a bit or two (obj might become jsonObject and so on). Yet it should become productive in the end

Without jsonSchema part we have to extract objects and make checks more strict, which is problematic for dates and IDs